app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -25,6 +26,11 @@ def index
 
     private
 
+    def check_sp_active
+      return if ServiceProvider.from_issuer(params[:client_id]).active?
```

Comment:
we should be able to do `@authorize_form.service_provider` here:

```suggestion
      return if @authorize_form.service_provider.active?
```

Reply:
Ah yeah, that makes sense. I was moving around when the action occurred a bunch and for a time it was before the form was created. I'll change it.